runhouse/resources/hardware/cluster.py

```diff
@@ -948,6 +953,9 @@ def restart_server(
             logs_level = DEFAULT_LOG_LEVEL
         logger.info(f"Setting cluster log level to: {logs_level}")
 
+        if not enable_telemetry_collection:
+            enable_telemetry_collection = rh.configs.telemetry_collection_enabled()
```

Comment:
confused about this - shouldn't it be disabled here?

Reply:
or maybe im confused by the variable names - we're checking here if telemetry is enabled if no argument is explicitly provided? if it's not explicitly provided why not just set the default to True?